tools/debug/protocols/sync/mod.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2020 Ant Financial
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+pub mod image;
+pub mod image_ttrpc;
+pub mod agent;
+pub mod agent_ttrpc;
+pub mod empty;
+pub mod health;
+pub mod health_ttrpc;
```

Comment:
I think these protocols(agent,empty,health etc) are for kata CC, they are not needed in the enclave cc scenario.